cmd/kyverno/main.go

```diff
@@ -392,6 +395,15 @@ func main() {
 		webhookCfg.UpdateWebhookChan <- true
 	}
 
+	if splitPolicyReport {
+		setupLog.Info("cleanup old policy reports", "splitPolicyReport", splitPolicyReport)
+		err = policyreport.CleanupPolicyReport(kyvernoClient)
+		if err != nil {
+			setupLog.Error(err, "failed to delete old reports")
+			os.Exit(1)
+		}
+	}
```

Comment:
Can we run this in a goroutine? What if this takes minutes to complete? Would Kyverno crash due to webhook checks?

Reply:
moved the cleanup logic in reportcontroller.

Reply:
Did you push the change?